admin-ui/app/locales/fr/translation.json

```diff
@@ -56,7 +56,7 @@
     "aliases": "Aliases",
     "trust_relationships": "Relations de confiance",
     "api_config": "Configurer les propriétés de l'API",
-    "builtIn": "Intégré",
+    "basic": "Basique",
```

Comment:
Using an inconsistent translation key (`builtIn` instead of `basic`) is bad because it violates the principle of consistency by creating a naming discrepancy for the same concept across different files, which increases cognitive load for developers and can lead to bugs if the wrong key is used.